docs/guides/run-elasticsearch-local.rst

```diff
@@ -0,0 +1,99 @@
+==========================================
+Enabling Elasticsearch on the local server
+==========================================
+
+Read the Docs has been using Elasticsearch which is a platform for distributed search and analysis of data in real time. To enable the search feature on your local installation, you need to install the elasticsearch locally and run the Elastic server. 
```

Comment:
Style nit: Please wrap lines at 80 characters.

Reply:
@rajujha373 @berkerpeksag actually, rtd doesn't use that style of wrapping all lines to 80 characters, here are the guidelines for writing docs https://docs.readthedocs.io/en/latest/docs.html#content. Note that this was recently added.